src/main/com/fulcrologic/fulcro/components.cljc

```diff
@@ -1181,9 +1181,13 @@
                queried-keywords      (-legal-keys template)
                has-wildcard?         (some #{'*} template)
                to-sym                (fn [k] (symbol (namespace k) (name k)))
-               illegal-syms          (mapv to-sym (set/difference destructured-keywords queried-keywords))]
+               illegal-syms          (mapv to-sym (set/difference destructured-keywords queried-keywords))
+               component-query       #(and (list? %) (= (name `get-query) (name (first %))) %)
```

Comment:
`(= 'query (name (first %)))` perhaps?

Reply:
I guess you meant replacing 
```
(name `get-query)
```
with `"get-query"`? I prefer to use the actual symbol instead of a hardcoded string so that if you ever rename it, it wouldn't break. But given how that is unlikely string is fine, if that is what you prefer.